ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -304,7 +315,7 @@ def _llmobs_set_meta_tags_from_similarity_search(
                 if isinstance(formatted_inputs, str):
                     span.set_tag_str(INPUT_VALUE, formatted_inputs)
                 else:
-                    span.set_tag_str(INPUT_VALUE, json.dumps(formatted_inputs))
+                    span.set_tag_str(INPUT_VALUE, json.dumps(formatted_inputs, default=_unserializable_default_repr))
```

Comment:
Is the lack of `skip_keys=True` on this one intentional?

I was thinking that maybe we should have a utility function to do the `json.dumps(..., skipkeys=True, default=_unserializable_default_repr)` and use it everywhere instead of repeating this at multiple places.

Reply:
Thanks for the catch + suggestion. I've made a utility helper following your suggestion, thanks!